backend/README.md

```diff
@@ -2,6 +2,53 @@
 
 ## Setup
 
+In order to have all the secret keys set up properly, you must create a `.env` file and a `keys.json` file to the root of this directory (`./backend`).
+
+Example setup:
+
+In `.env`:
+```
+MONGO_UURL=mongodb://mongouser:mykey@somedb.mlab.com:port/db
+MONGO_DURL=mongodb://mongouser:mykey@somedb.mlab.com:port/db
+MONGO_URL=mongodb://mongouser:mykey@somedb.mlab.com:port/db
+LEAD_SUFFIX=l
+DIRECTOR_SUFFIX=d
+KEY_JSON=../../keys.json
+SCHEDULER_URL=https://scheduler-URI.amazonaws.com/endpoint
+SCHEDULER_API_KEY=abcd1234
+```
+
+In `keys.json`:
+```
+{
+  "keys": [
+    {
+      "name": "Director 1",
+      "key": "1234"
+    }
+  ]
+}
+```
+
+### Docker
+
+To run with [Docker](https://www.docker.com/) (and start the frontend and backend all at once with one command):
+```
+docker-compose up
+```
+
+And when you're done, you can run:
+```
+docker-compose down -v
+```
+
+Note that if there's a change in the `package.json` file (such as a new module added into the repository), instead of running `yarn add` to re-install, just run `docker-compose up --build`. 
```

Comment:
didn't know about this!

Reply:
Yeah! Essentially I'm mounting the `./backend` and `./frontend` directories to the container for easy development, but then mounting on top of that a named volume, `fe_modules` and `be_modules` for both the frontend and backend respectively. So the process goes like this:

1. User creates Docker images for FE and BE, each with `node_modules` installed.
2. Upon starting Compose, if `fe_modules` or `be_modules` hasn't yet been created, it will grab those from the *image* and copy them to the host.
3. It will mount both the frontend and backend folders in the host to the corresponding folders in the container. This allows changes locally in code to be reflected immediately in the container.
4. It will mount over that the copied `fe_modules` and `be_modules` folders. This means that it will overwrite any local `node_modules` and replace with the copy from the image. 

This ensures uniformity on module builds among all our devs using Docker! Ta-da! :)